inst/htmlwidgets/datatables.js

```diff
@@ -1081,6 +1081,24 @@ HTMLWidgets.widget({
       table.draw();
     }
 
+    methods.hideCols = function(hide, reset) {
+      if (reset) {
+        $(table.columns().visible(true, false))
```

Comment:
@yihui I think `table.columns().visible(true, false);` should be ok. Am I right?

Reply:
Good catch. You are correct. There is no need to use `$()`.

Reply:
```js
methods.showCols = function(show, reset) {
      if (reset) table.columns().visible(false, false);
      table.columns(show).visible(true);
}
```
I think @yihui will like this style but I can't edit the PR directly.

Reply:
Right. Usually I don't use curly braces if there is only a single expression in them, but I'm fine with them.